app/src/main/kotlin/com/wire/android/ui/home/HomeDrawer.kt

```diff
@@ -111,6 +120,22 @@ fun HomeDrawer(
             )
         }
 
+        DrawerItem(
+            data = ReportBug.getDrawerData(),
+            selected = false,
+            onItemClick = {
+
+                val dir = File(viewModel.logFilePath()).parentFile
+                val logsUris = context.getUrisOfFilesInDirectory(dir)
+                val intent = context.multipleFileSharingIntent(logsUris)
+                intent.putExtra(Intent.EXTRA_EMAIL, arrayOf("wire-newandroid@wearezeta.zendesk.com"))
+                intent.putExtra(Intent.EXTRA_SUBJECT, "Bug Report - Wire Beta")
+                intent.putExtra(Intent.EXTRA_TEXT, viewModel.reportBugEmailTemplate(context.getDeviceId()?.sha256()))
+                intent.type = "message/rfc822"
+
+                context.startActivity(Intent.createChooser(intent,"Choose an Email client : "))
```

Comment:
Maybe we can extract this to a string resource as well ?  `Choose an Email client` 🤔

Reply:
when we localise it then yes. I am not sure this poor man's reporting button will stay after beta